src/core/reopt_inputs.jl

```diff
@@ -353,7 +353,7 @@ function setup_tech_inputs(s::AbstractScenario, time_steps)
     cap_cost_slope = Dict{String, Any}()
     om_cost_per_kw = Dict(t => 0.0 for t in techs.all)
     production_factor = DenseAxisArray{Float64}(undef, techs.all, 1:length(s.electric_load.loads_kw))
-    tech_renewable_energy_fraction = Dict(t => 1.0 for t in techs.all)
+    tech_renewable_energy_fraction = Dict(t => 1.0 for t in union(techs.elec, techs.fuel_burning))
```

Comment:
@zolanaj we still don't have a "tech_renewable_energy_fraction" for Boiler. I'm not suggesting we implement that in this PR, but do you think it would be safer to default this to 0 (like we do in bau_inputs) rather than 1 for cases like this in which we haven't yet implemented that input?

Reply:
Oh I'm realizing if we do that we need to explicitly define Wind and PV as tech_renewable_energy_fraction = 1.0. So I guess either: 
- Default it to zero and define PV and Wind as 1 within their setup_tech functions
- Or leave the default as 1 but temporarily define the boiler tech_renewable_energy_fraction as 0 in its setup_tech function
?

Reply:
> @zolanaj we still don't have a "tech_renewable_energy_fraction" for Boiler. I'm not suggesting we implement that in this PR, but do you think it would be safer to default this to 0 (like we do in bau_inputs) rather than 1 for cases like this in which we haven't yet implemented that input?

I think it's worth adding, so I've implemented it here: https://github.com/NREL/REopt.jl/commit/0ce4f171743fd61ca842137dc0d3821081336039

I'll follow up with an update to the changelog.

Reply:
> Oh I'm realizing if we do that we need to explicitly define Wind and PV as tech_renewable_energy_fraction = 1.0. So I guess either:
> 
> * Default it to zero and define PV and Wind as 1 within their setup_tech functions
> * Or leave the default as 1 but temporarily define the boiler tech_renewable_energy_fraction as 0 in its setup_tech function
>   ?

I think that it's probably best to not include any starting assumptions and change things one technology at a time.  This is implemented here: https://github.com/NREL/REopt.jl/commit/03d5f383dc069b694dd51d50cbabb3bde41af069

Reply:
This fix is great!